.bazelrc

```diff
@@ -13,7 +13,6 @@ build:linux --extra_toolchains=@llvm_toolchain//:cc-toolchain-x86_64-linux
 
 build:no-warnings --per_file_copt=//.*@-Werror
 
-common:linux:ci --config=no-warnings
```

Comment:
doubling up these configs with `:config1:config2` doesn't work. I don't know why I thought it did.